Sapphire dashboards/Analytics performance.json

```diff
@@ -0,0 +1,3207 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 2,
+  "id": 5,
+  "iteration": 1638902658553,
+  "links": [],
+  "panels": [
+    {
+      "datasource": null,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 31,
+      "title": "Row title",
+      "type": "row"
+    },
+    {
+      "datasource": "Timescale",
```

Comment:
Interesting.  I was surprised to find the Timescale datasource being used.  Are we using TimescaleDB in cClear?  (If we are, where do I go for more information?)
I found one document mentioning TimescaleDB on Confluence.  A quick glance seems to indicate that TimescaleDB is used within Sapphire: https://cpacketnetworks.atlassian.net/wiki/spaces/CS/pages/866484233/Running+capture+container.  Is the idea that the dashboard will pull data from the TimescaleDB container running on Sapphire?  If this is the case, then we'll need to include instructions for setting up the Timescale datasource.